app/javascript/components/DropdownSelector.js

```diff
@@ -171,40 +189,62 @@ const DropdownSelector = ({ sectors, companies, selectedOption, defaultFilter =
             <div className="dropdown-selector__options-wrapper">
               {isFilterBySector && (
                 <div className="dropdown-selector__options">
-                  {options.map(option => (
-                    <div onClick={() => handleOptionClick(option)} className="dropdown-selector__option">
+                  {(options.length > 0 && options.map((option, i) => (
+                    <div
+                      onClick={() => handleOptionClick(option)}
+                      className="dropdown-selector__option"
+                      key={`${option.name}-${i}`}
+                    >
                       {option.name}
                     </div>
+                  ))) || (searchValue.length > 0 && options.length === 0 && (
```

Comment:
same here: you can remove `> 0` bit and instead of explicitly checking if `options.length === 0` turn it into `!options.length`

Reply:
Same motif as in the comment before.